pimd/pim_nht.c

```diff
@@ -898,7 +898,7 @@ int pim_ecmp_nexthop_lookup(struct pim_instance *pim,
 	uint32_t num_nbrs = 0;
 	char addr_str[PREFIX_STRLEN];
 
-	if (PIM_DEBUG_PIM_NHT) {
+	if (PIM_DEBUG_PIM_NHT_DETAIL) {
 		pim_inet4_dump("<addr?>", src->u.prefix4, addr_str,
```

Comment:
Just a quick question regarding this:

![Screenshot from 2021-12-02 09-39-01](https://user-images.githubusercontent.com/3352707/144378436-ac208685-ec75-4a88-bcdb-fc7ec2b02134.png)

If PIM_DEBUG_PIM_NHT_DETAIL is turned on, then `addr_str` is filled, right? But below `addr_str` is used only if PIM_DEBUG_PIM_NHT is enabled? Won't addr_str just be a garbage there?